services/wallet/collectibles/ownership_db.go

```diff
@@ -396,6 +398,61 @@ func (o *OwnershipDB) Update(chainID w_common.ChainID, ownerAddress common.Addre
 	return
 }
 
+func (o *OwnershipDB) FetchCachedCollectibleOwnersByContractAddress(chainID w_common.ChainID, contractAddress common.Address) (*thirdparty.CollectibleContractOwnership, error) {
```

Comment:
sure, please note that this is exactly the same behavior as for `FetchBalancesByOwnerAndContractAddress`. Instead of looking on chain we just get the data in cache, but let me have a look if I can add a test that makes sense if not already implemented somewhere.